src/problems/127.word-ladder.jl

```diff
@@ -0,0 +1,96 @@
+#
+# @lc app=leetcode id=127 lang=julia
+#
+# [127] Word Ladder
+#=
+Given two words (beginWord and endWord), and a dictionary's word list, find the length of shortest transformation sequence from beginWord to endWord, such that:
+
+Only one letter can be changed at a time.
+Each transformed word must exist in the word list.
+Note:
+
+Return 0 if there is no such transformation sequence.
+All words have the same length.
+All words contain only lowercase alphabetic characters.
+You may assume no duplicates in the word list.
+You may assume beginWord and endWord are non-empty and are not the same.
+Example 1:
+
+Input:
+beginWord = "hit",
+endWord = "cog",
+wordList = ["hot","dot","dog","lot","log","cog"]
+
+Output: 5
+
+Explanation: As one shortest transformation is "hit" -> "hot" -> "dot" -> "dog" -> "cog",
+return its length 5.
+Example 2:
+
+Input:
+beginWord = "hit"
+endWord = "cog"
+wordList = ["hot","dot","dog","lot","log"]
+
+Output: 0
+
+Explanation: The endWord "cog" is not in wordList, therefore no possible transformation.
+=#
+# import Pkg
+# Pkg.add("DataStructures")
+using DataStructures
+# @lc code=start
+function ladderLength(beginWord::String, endWord::String, wordList::Vector{String})::Int
```

Comment:
Let's stick with the BlueStyle to use lower case with underscors for method names. https://github.com/invenia/BlueStyle#synopsis